x-pack/plugins/infra/public/components/asset_details/tabs/overview/metadata_summary/metadata_summary_list.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import type { InfraMetadata } from '../../../../../../common/http_api';
+import { NOT_AVAILABLE_LABEL } from '../../../translations';
+import { useTabSwitcherContext } from '../../../hooks/use_tab_switcher';
+import { FlyoutTabIds } from '../../../types';
+import { ExpandableContent } from '../../../components/expandable_content';
+import { MetadataHeader } from './metadata_header';
+
+interface MetadataSummaryProps {
+  metadata: InfraMetadata | null;
+  metadataLoading: boolean;
+}
+
+export interface MetadataData {
+  field: string;
+  value: string | string[] | undefined;
+  tooltipFieldLabel: string;
+  tooltipLink?: string;
+}
+
+const metadataData = (metadataInfo: InfraMetadata['info']): MetadataData[] => [
+  {
+    field: 'hostIp',
+    value: metadataInfo?.host?.ip,
+    tooltipFieldLabel: 'host.ip',
+    tooltipLink: 'https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip',
+  },
+  {
+    field: 'hostOsVersion',
+    value: metadataInfo?.host?.os?.version,
+    tooltipFieldLabel: 'host.os.version',
+  },
+];
+
+export const MetadataSummaryList = ({ metadata, metadataLoading }: MetadataSummaryProps) => {
+  const { showTab } = useTabSwitcherContext();
+
+  const onClick = () => {
+    showTab(FlyoutTabIds.METADATA);
+  };
+
+  return (
+    <EuiFlexGroup gutterSize="m" responsive={false} wrap justifyContent="spaceBetween">
+      <EuiFlexGroup alignItems="flexStart">
+        {metadataData(metadata?.info).map(
+          (metadataValue) =>
+            metadataValue && (
```

Comment:
Q: when `metadatValue` is `undefined` or `null` does it mean it is still loading? Should we show a loading indicator in this case?

Reply:
We already show the spinner inside the value section (line 69), it would look strange to have another one here